### PR TITLE
Add rewrite_polyfills to custom builds tutorial

### PR DESCRIPTION
**Pull request by probins**
Custom builds tutorial got left out of the recent compiler update PR, so this amends it. While I'm at it, I've also added use_types_for_optimization, which will help reduce build size.

### CODE REVIEW

File: doc/tutorials/custom-builds.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ Creating a custom build requires writing a build configuration file. The format
       "api", "observable"
     ],
     "compilation_level": "ADVANCED",
-    "manage_closure_dependencies": true
+    "manage_closure_dependencies": true,
+    "use_types_for_optimization": true,
```
Comment (gberaudo) [inline doc/tutorials/custom-builds.md]: This is the default, see https://github.com/google/closure-compiler/wiki/FAQ#some-of-my-properties-are-getting-renamed-but-some-arent-why.

**Comment by gberaudo** [comment]
Thanks for spotting this @probins.

File: doc/tutorials/custom-builds.md (hunk 1 above)
Comment (probins) [inline doc/tutorials/custom-builds.md]: yeah, you're right. I'll amend this PR and remove this

**Comment by probins** [comment]
ok, removed references to `use_types_for_optimization` (I can also remove it from my own configs :-))

**Comment by fredj** [comment]
thanks @probins

**Pull request merged**